portfolio/src/main/java/com/google/sps/servlets/DeleteServlet.java

```diff
@@ -0,0 +1,34 @@
+package com.google.sps.servlets;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
```

Comment:
nit: blank line to separate static and non-static imports; same for DataServlet.java (I can't comment on that unedited line for some reason)